controllers/CommentsController.js

```diff
@@ -0,0 +1,130 @@
+import db from '../models';
+import utils from '../helpers/utilities';
+import validateComment from '../helpers/validateComment';
+
+const { Comment, User, Reply } = db;
+
+/** * Class representing comments */
+export default class CommentsController {
+  /**
+   * @function createComment
+   * @summary Allows a user create comment
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @returns {object} status code 201 with comment created if successful
+   */
+  static createComment(req, res) {
+    const validator = validateComment(req);
+    if (validator.length !== 0) {
+      return res.status(403).json({
+        success: false,
+        errors: {
+          body: [...validator]
+        }
+      });
+    }
+    const { body } = req.body.comment;
+    Comment.create({
+      userId: req.userId,
+      articleId: req.articleObject.id,
+      body
+    })
+      .then(comment => res.status(201).json({
+        success: true,
+        comment,
+        user: utils.userToJson(req.userObject),
+        article: req.articleObject,
+      }))
+      .catch(err => res.status(500).json({
+        success: false,
+        errors: {
+          body: [err.message]
+        },
+      }));
+  }
+
+  /**
+   * @function createReply
+   * @summary Allows a user create comment
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @returns {object} status code 201 with comment created if successful
+   */
+  static createReply(req, res) {
+    const validator = validateComment(req);
```

Comment:
Sammy Sammy! great job. I will suggest you use  `validateComent` as a middleware for the reply and comment route just to prevent getting to the controllers if the request object fails validation.